src/govpress-unit/code-review.md

```diff
@@ -0,0 +1,88 @@
+
+In particular, if you are requesting review, you should be aware that reviewers will be reviewing multiple changesets over a large number of very different projects each day, and will not necessarily be aware of the full context of each story. This makes it especially important that we follow a consistent and efficient process.
+
+This document should therefore be read _in addition_ to the [Tech Team documentation](https://playbook.dxw.com/tech/how-we-do-development/) and applies to all PRs that are not automatically generated by services such as Dependabot or GovPress Tools.
+
+## Creating PRs
+
+When you create a PR the **Description** section of the PR should include, as a minimum:
+
+1. A full description of the story that the PR addresses. For example:
+    * [nao/353](https://github.com/dxw/nao/pull/353)
+    * [whippet/212](https://github.com/dxw/whippet/pull/212)
+    * [website/359](https://github.com/dxw/website/pull/359)
+1. Links to any relevant external documentation, which might be internal documentation (Zendesk, Trello, etc.) or external documentation. For example:
+    * [website/286](https://github.com/dxw/website/pull/286)
+    * [valleys-to-coast/101](https://github.com/dxw/valleys-to-coast/pull/101)
+    * [govpress-tools/122](https://github.com/dxw/govpress-tools/pull/122)
+1. Details of any concerns that might not be obvious from the code (these can usually be copied from commit messages). For example:
+    * [dalmatian/543](https://github.com/dxw/dalmatian/pull/543)
+1. Full instructions on how the reviewer can test the PR locally (or in some other appropriate way), and what results the reviewer can expect. This section should **never be skipped** when making changes to source code. If the PR will change how a site looks, please include screenshots to help to illustrate the expected changes. For example:
+    * [lambethtogeher/160](https://github.com/dxw/lambethtogether/pull/160)
+    * [govpress-product-theme/167](https://github.com/dxw/govpress-product-theme/pull/167)
+
+If your PR interacts with the database or contains any front-end code, special consideration should be given to security concerns. If you are not sure what security considerations you need to be aware of, please ask a Tech Lead before pushing your changes.
+
+## Reviewing PRs
+
+All developers are expected to undertake code review. Unless a PR only includes documentation, or similar, you should _always_ test the changeset locally. If it is not clear how to do this, ask the PR author using the **Request changes** feature in GitHub, or similar.
+
+PRs should not be held up unnecessarily, so if you have questions about the changeset, or small requests, be clear to tell the author if your comments are non-blocking. Your main role is to prevent bugs appearing on live systems, and to minimise the amount of technical debt in our repositories. If your suggested changes do not achieve either of those aims, they should not block a merge.
```

Comment:
> PRs should not be held up unnecessarily, so if you have questions about the changeset, or small requests, be clear to tell the author if your comments are non-blocking. Your main role is to prevent bugs appearing on live systems, and to minimise the amount of technical debt in our repositories. If your suggested changes do not achieve either of those aims, they should not block a merge.

Feels relevant in the general tech team guidance too